tests/baselines/reference/jsFileCompilationMethodOverloadSyntax.errors.txt

```diff
@@ -0,0 +1,14 @@
+error TS5055: Cannot write file 'tests/cases/compiler/a.js' because it would overwrite input file.

```

Comment:
Is this supposed to happen?

Reply:
Yes, all the JS tests have this error. https://github.com/search?q=repo%3Amicrosoft%2FTypeScript%20error%20TS5055%3A%20Cannot%20write%20file%20%27tests%2Fcases%2Fcompiler%2Fa.js%27%20because%20it%20would%20overwrite%20input%20file.&type=code

Reply:
We typically set outDir or noEmit to silence these, FWIW

Reply:
Okay, I suppose I can do that...